### DISCUSSION TITLE
sequence order of codes when using custom commands

### DISCUSSION THREAD

**Discussion by pappurvsa** [visose/Robots#89] [Q&A] [answered]
While using custom commands the program/ code generator loses the sequence. 
![191120_RobotsProblem](https://user-images.githubusercontent.com/8084742/69240449-67a2a880-0b94-11ea-9b57-81864cc1e0c7.png)

**Comment by visose** [comment, marked as answer]
The C input is for the command statements. The D input is for variable declarations, which appear at the top of the code.
